[Scenario 2] Change size of the "broken" image to be 0 bytes

slawqo opened the pull request:
Guides to that scenario says about checking size of the image in Glance, previously when it was simply empty file, size was 0 and now it should still be 0 bytes.

cubeek commented:
I created this scenario because I experienced first hand that I did "curl https://some-cirros-url" with no link redirection - so it actually downloaded the HTML that redirects the request in a browser. That means the HTML file was not 0 bytes - perhaps we should rather upgrade the guide to make sure the downloaded image is a valid qcow image format instead?

slawqo commented:
I updated guided solution for this scenario and removed points related to checking image size - you're right that this is not really needed now. You can check it in https://redhatquickcourses.github.io/rhosp-bfx-020/rhosp-bfx019/1/chapter2/section2.html - please let me know if that is ok for you.
